packages/zowe-explorer/src/utils/LoggerUtils.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the *
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at *
+ * https://www.eclipse.org/legal/epl-v20.html                                      *
+ *                                                                                 *
+ * SPDX-License-Identifier: EPL-2.0                                                *
+ *                                                                                 *
+ * Copyright Contributors to the Zowe Project.                                     *
+ *                                                                                 *
+ */
+
+import { Gui } from "@zowe/zowe-explorer-api";
+import * as vscode from "vscode";
+import * as nls from "vscode-nls";
+import * as globals from "../globals";
+
+// Set up localization
+nls.config({
+    messageFormat: nls.MessageFormat.bundle,
+    bundleFormat: nls.BundleFormat.standalone,
+})();
+const localize: nls.LocalizeFunc = nls.loadMessageBundle();
+
+export async function initializeZoweLogger(context: vscode.ExtensionContext): Promise<void> {
```

Comment:
Same scenario as `ProfileUtils.ts` - not asking you to make any changes, just thinking out loud for future codebase refactoring 😅

Reply:
I definitely like the idea of a singleton or class-with-static methods 😋